CMakeLists.txt

```diff
@@ -5,6 +5,16 @@ cmake_minimum_required(VERSION 3.22)
 set(CMAKE_TRY_COMPILE_TARGET_TYPE STATIC_LIBRARY)
 project(msvc_standard_libraries LANGUAGES CXX)
 
+include(CheckCXXSourceCompiles)

```

Comment:
Curious why this approach? Wouldn't `find_package(WindowsSDK)` work? And, it also can tell which directory was found etc.?

https://github.com/rpavlik/cmake-modules/blob/main/FindWindowsSDK.cmake

Reply:
This would be depending on a full find module for, in my opinion, no real reason. This works and gives us the result we want.

Reply:
Yeah, it would be one thing if it was a builtin module, but it looks like it is not